ext/com_dotnet/com_saproxy.c

```diff
@@ -365,7 +373,9 @@ static void saproxy_free_storage(zend_object *object)
 //???		}
 //???	}
 
-	OBJ_RELEASE(&proxy->obj->zo);
+	if (&proxy->obj->zo != NULL) {
```

Comment:
I suppose you intended to write:
```suggestion
	if (proxy->obj != NULL) {
```
Otherwise this is UB.

Reply:
Oops. Thank you!